app/static/src/app/components/options/NumericInput.vue

```diff
@@ -39,11 +49,21 @@ export default defineComponent({
         // this on every keystroke, so update text value on blur, mounted, and external value change (a change
         // to a value which wasn't the last one changed to here e.g. caused by model fit)
         const lastNumericValueSet = ref<null|number>(null);
+        const validatedNumeric = ref<null|number>(null);
         const textValue = ref("");
 
-        const formatTextValue = () => {
+        const errorTooltipProps: ToolTipContent = {
```

Comment:
..and on the other hand I'm surprised that this doesn't need this to be reactive. My instinct would have been to make the 'tooltipContent' string a ref, and build a computed value for the whole props object.  This seems to work though!

Reply:
yh i think it is because the input component is binded to textvalue which always updates, triggering update of the input component so the directive gets its refresh